### DISCUSSION TITLE
[Question]: What are screenshots on the main page of?

### DISCUSSION THREAD

**Discussion by aik** [blakeblackshear/frigate#12403] [Ask A Question]
### What is your question:

I am new to frigate. There are 4 screenshots on https://github.com/blakeblackshear/frigate and it is not webbrowser or HASS, what is it? Thanks,

**Comment by didi3r** [comment]
They are a screenshot from frigate 0.14.

**Comment by blakeblackshear** [comment]
Since 0.14 is a PWA, you can install it as an app in Chrome with the "Install" icon on the right side of the URL bar. This is from macOS.
![image](https://github.com/user-attachments/assets/886323a8-9780-48c9-9d95-f32177a570ee)
